Remotion/Web/Development.WebTesting/Utilities/AssertionExceptionUtility.cs

```diff
@@ -30,7 +30,7 @@ public static class AssertionExceptionUtility
     [MustUseReturnValue]
     public static Exception CreateControlDisabledException ()

```

Comment:
Ah i see that we where using Exception in this method, this is why you used that too instead of the specific. No idea why we did that, check with MK and then it can probably be changed in the whole class to WebTestException

Reply:
We should check with MK. I guess it would make no difference here.